plugins/python/io/NumpyReader.cpp

```diff
@@ -147,6 +166,34 @@ PyArrayObject* load_npy(std::string const& filename)
     return reinterpret_cast<PyArrayObject*>(array);
 }
 
+PyArrayObject* load_npy_script(std::string const& source,
+                               std::string const& module,
+                               std::string const& function,
+                               std::string const& fargs)
+{
+
+    MetadataNode m;
+    std::unique_ptr<plang::Script> script(new plang::Script(source, module, function));
```

Comment:
Is there some reason for these not just to be on the stack?

Reply:
addressed by fddec900a5c59978842bd541e53784e578215084